tool/doc.dart

```diff
@@ -137,8 +156,9 @@ Future<void> fetchBadgeInfo() async {
   }
 
   var flutter = await fetchConfig(
-      'https://raw.githubusercontent.com/flutter/flutter/master/packages/flutter/lib/analysis_options_user.yaml');
+      'https://raw.githubusercontent.com/flutter/packages/master/packages/flutter_lints/lib/flutter.yaml');
   if (flutter != null) {
+    flutterRules.addAll(recommendedRules);
```

Comment:
What about the core rules?

Or should they be added to the recommendedRules above instead?

Reply:
Good catch.  I meant to do that but didn't sanity check!

Reply:
Fixed.